Engine.Core/DataStore.fs

```diff
@@ -26,7 +26,17 @@ type DataStore() =
         member __.TestFailureInfoUpdated : IEvent<_> = testFailureInfoUpdated.Publish
         member __.CoverageInfoUpdated : IEvent<_> = coverageInfoUpdated.Publish
         member __.UpdateRunStartParams(rsp : RunStartParams) : unit = runStartParams <- rsp |> Some
-        
+        member __.ResetData() = 
+            sequencePoints <- new PerDocumentSequencePoints()

```

Comment:
Can't remove the dry violation here. Want to remove the duplication by calling Update(), but f# syntax is not letting me. Do you have a suggestion?